pkg/features/kube_features.go

```diff
@@ -761,24 +761,6 @@ const (
 	// Subdivide the ClusterIP range for dynamic and static IP allocation.
 	ServiceIPStaticSubrange featuregate.Feature = "ServiceIPStaticSubrange"
 
-	// owner: @andrewsykim @uablrek
```

Comment:
FYI @uablrek

Reply:
@andrewsykim Thanks, and welcome back! :smile: